functional/basic-attestation-on-localhost/test.sh

```diff
@@ -27,6 +29,11 @@ rlJournalStart
         if [ -n "$KEYLIME_TEST_DISABLE_REVOCATION" ]; then
             rlRun "limeUpdateConf cloud_agent listen_notifications False"
         fi
+        #FIXME different parsing of elements in action list in python agent
+        if limeIsPythonAgent; then
```

Comment:
This function is not yet available in the main branch. Maybe we should add it there...WDYT?

Reply:
Yes, I know it. We will wait until you add it. Or should I add it as standalone commit? @kkaarreell